configure.ac

```diff
@@ -620,6 +620,14 @@ AC_PROG_LEX([noyywrap])
 AX_PROG_FLEX
 AX_PROG_BISON
 
+AC_CACHE_CHECK([for diff], [ac_cv_path_DIFF],
+  [AC_PATH_PROGS_FEATURE_CHECK([DIFF], [diff],
+    [[$ac_path_DIFF --version >/dev/null
```

Comment:
Shouldn't there be a check for return code zero/nonzero with two files instead of --version which may not be supported in some non-gnu systems?